.vagrant/machines/default/virtualbox/private_key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Woah! @rheaSPK I'd really suggest you to take care of your credentials and reset them now ...

Reply:
Also, please update your global `.gitignore` according to https://github.com/hpi-swt2/connections-portal/pull/56#issuecomment-738951815.